### DISCUSSION TITLE
What is the suggested way to record prefixes used across sites/locations/regions?

### DISCUSSION THREAD

**Discussion by firstkevinds** [netbox-community/netbox#6814] [Q&A] [answered]
Have many /30 networks PtP between physical locations.

Example:

Prefix - 192.0.2.32/30
SiteA - 192.0.2.33/30
SiteB - 192.0.2.34/30

Prefix - 192.0.2.36/30
SiteA - 192.0.2.37/30
SiteC - 192.0.2.38/30

What is the suggested way to enter this into NetBox?

**Comment by nahun** [comment, marked as answer]
Its up to you in the end. Most of our peerings are done from a data center to a remote office so we set the NetBox Site to the data center (but I could easily see people wanting to do the opposite). When its DC to DC, we typically choose the DC with the bigger footprint.

Maybe you could follow routing? If the default route goes to SiteA, assign that site. Obviously there might not be a default route, but you get the idea...

**Reply by firstkevinds** [reply]
> Maybe you could follow routing? If the default route goes to SiteA, assign that site. Obviously there might not be a default route, but you get the idea...

To expand my example,

Prefix - 192.0.2.32/30
SiteA - 192.0.2.33/30
SiteB - 192.0.2.34/30

Prefix - 192.0.2.36/30
SiteA - 192.0.2.37/30
SiteC - 192.0.2.38/30

SiteA - 198.51.100.0/24
SiteB - 203.0.113.0/24

SiteA's router routes traffic for 203.0.113.0/24 to 192.0.2.34.  SiteB's router routes traffic for 198.51.100.0/24 to 192.0.2.33.

But it sounds like this scenario isn't been covered in the software?

So what I am thinking is creating a Point-to-Point 'site' putting them in there..  Maybe a feature suggestion to allow multiple sites for a prefix?

So there was a feature request for something similar already So there was a feature request for this already  https://github.com/netbox-community/netbox/issues/6414  I'm thinking making Sites 'tags' then multiple sites could be attached.
